.docker/docker-compose.yml

```diff
@@ -4,6 +4,12 @@ services:
   app:
     build:
       context: ../
+    environment:
+      DATABASE_NAME: article-store
+      DATABASE_USER: user
+      DATABASE_PASSWORD: secret
+      DATABASE_HOST: db
+      DATABASE_PORT: 5432
```

Comment:
saw a comment on connection strings. Advantages of multiple environment variables here:

- the more single environment variables can have sane defaults, the better to separate them rather than having a single string as it's easier to override just one or two of them. In the case of database connections though, the only variable with a default is `DATABASE_PORT`.
- the format of these variables is easier to get than a connection string (by memory: `postgres://user:pass@host:port/databasename`? Not sure if I got that right whereas I know I have with single variables)

Advantages of connection strings:

- if the application/library supports it, they can provide abstraction of the database driver as well e.g. passing in `sqlite://:memory:` rather than `postgres://...`
- fewer variables to document

For both cases, I wonder if there is any standard on the naming. This is Postgres-specific so the [`PG*` variable names](https://www.postgresql.org/docs/current/libpq-envars.html) could be used? Unless someone would think they will be magically picked up (like the `AWS_*` ones) rather than read by the application.